base_phone/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========
+Base Phone
+==========
+
+This module validate phone numbers using the *phonenumbers* Python library,
```

Comment:
this text is not the same as the description.rst file. It will be lost, when merging the PR.
could you edit / add the files present in readme folder ? 

adding contributors, develop, etc... (and optionally screen shot.)

thanks !

Reply:
The up-to-date version is in the readme subfolder. The README.rst will be re-generated from the readme subfolder. I don't know how to regenerate the README.rst from the subfolder by hand.